src/event/handlers.ts

```diff
@@ -0,0 +1,23 @@
+import type { EventHandlers } from "../../deps.ts";
+import type { HandlerOptions } from "../../shared.ts";
+
+import { createEventHandlers } from "../../deps.ts";
+
+// Load handlers statically.
+import {
+  guildCreateHandler,
+  interactionCreateHandler,
+  messageCreateHandler,
+  readyHandler,
+} from "./handlers/mod.ts";
+
+const eventHandlers = (options: HandlerOptions): Partial<EventHandlers> => {
```

Comment:
HandlerOptions has several properties which are used on generating handler functions.
CommandManager instance is one of such properties.